lib/string.c

```diff
@@ -43,6 +43,7 @@ int memcmp(const void *s1, const void *s2, size_t n)
     return 0;
 }
 
+#if 0
 void *memcpy(void *dest, const void *src, size_t n)
```

Comment:
If we don't use our own function anymore, it's probably better to remove it completely.

Reply:
Fine with me :)
I'll do it later.